### PR TITLE
OPSEXP-2445 Add guide to deploy dedicated hazelcast cluster with alfresco content services chart. 

### PR DESCRIPTION
**Pull request by pmacius**
OPSEXP-2445

### CODE REVIEW

**Review by gionn — CHANGES_REQUESTED** [review]
reformat to match 80 width column (there is a rewrap ext for vscode)

File: docs/helm/examples/external-hazelcast.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+
+1. Deploy alfresco-content-services. See [desktop
+   deployment](../desktop-deployment.md) section. Remember to stick to one
+   namespace in next steps.
+
+2. Prepare a valid Hazelcast xml configuration. First step in this [Alfresco
+   docs](https://docs.alfresco.com/content-services/latest/admin/cluster/#set-up-repository-clustering-via-external-hazelcast).
+   If you want to use config generated by us skip to 7th step.
+    - Example file:
+      [caches.properties](https://github.com/Alfresco/alfresco-community-repo/blob/master/repository/src/main/resources/alfresco/caches.properties)
```
Comment (gionn) [inline docs/helm/examples/external-hazelcast.md:37]:
```suggestion
2. Prepare a valid Hazelcast xml configuration, which also must include different cache
   definitions depending on the ACS version. You can generate it by following the steps
   in [Alfresco docs](https://docs.alfresco.com/content-services/latest/admin/cluster/#set-up-repository-clustering-via-external-hazelcast).
   For testing you can use config generated by us on 23.2.0-M1 and skip to 7th step.
    - Example file:
      [caches.properties](https://github.com/Alfresco/alfresco-community-repo/blob/master/repository/src/main/resources/alfresco/caches.properties)
```

File: docs/helm/examples/external-hazelcast.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+            </kubernetes>
+        </join>
+        <rest-api enabled="true">
+            <endpoint-group name="CLUSTER_READ" enabled="true"/>
+            <endpoint-group name="CLUSTER_WRITE" enabled="true"/>
+            <endpoint-group name="HEALTH_CHECK" enabled="true"/>
+            <endpoint-group name="HOT_RESTART" enabled="true"/>
+            <endpoint-group name="WAN" enabled="true"/>
+            <endpoint-group name="DATA" enabled="true"/>
+        </rest-api>
```
Comment (gionn) [inline docs/helm/examples/external-hazelcast.md:63]: didn't check, but those are necessary for management-center?

Comment (pmacius) [reply docs/helm/examples/external-hazelcast.md:63]: DATA one is necessary for manual testing for sure but the rest I will check if they influence mancenter.

File: docs/helm/examples/external-hazelcast_files/hazelcast.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+hazelcast:
+  javaOpts: -Dhazelcast.config=/data/hazelcast/hazelcast.xml
+  existingConfigMap: manual-hazelcast-configuration
+mancenter:
+  enabled: true
+  yaml:
+    hazelcast-client:
+      cluster-name: test
```
[the remark this answers is not in the document]
Comment (pmacius) [reply docs/helm/examples/external-hazelcast_files/hazelcast.yaml:8]: Mancenter has its own configmap: https://github.com/hazelcast/charts/blob/master/stable/hazelcast/values.yaml#L365

File: docs/helm/examples/external-hazelcast.md (hunk 2 above)
Comment (pmacius) [reply docs/helm/examples/external-hazelcast.md:63]: I left only DATA and mancenter is looking all right I guess we can keep DATA and delete rest.

File: docs/helm/examples/external-hazelcast.md [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+    helm repo add open https://simonmisencik.github.io/helm-charts
+    helm repo update
+    helm install testing open/ubuntu
+    ```
+
+3. Get into a shell of container created by previous commands.
+
+    ```bash
+    kubectl exec --stdin --tty testing-ubuntu -- /bin/bash
+    ```
```
Comment (gionn) [inline docs/helm/examples/external-hazelcast.md:165]: better use standard and well known stuff

no need for ubuntu helm chart, just use:

```
kubectl run -i --tty --rm --image ubuntu test-shell -- /bin/bash
```

**Review by gionn — CHANGES_REQUESTED** [review]

File: docs/helm/examples/external-hazelcast.md [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+5. Accordingly change the values for management center. Put internal network
+   address of your k8s.
+
+    ```xml
+    <management-center data-access-enabled="true">
+        <trusted-interfaces>
+            <interface>10.42.0.*</interface>
+        </trusted-interfaces>
+    </management-center>
+    ```
```
[the remark this answers is not in the document]
Comment (pmacius) [reply docs/helm/examples/external-hazelcast.md:81]: It comes with generated xml, I will mention to delete this

**Review by gionn — APPROVED** [review]

**Pull request merged**